README.md

```diff
@@ -8,11 +8,31 @@
 
 Here are the steps to follow to get going as an ADO content contributor (or heck, even a code contributor)
 
-1. Install [Hugo](http://gohugo.io) - *NOTE: Please make sure it is at least version 0.58.3*
+You can use VSCode + Remote extensions (and Docker), or you can use Docker outside of VSCode, or
+you can install hugo on your computer.
+
+### option 1: VSCode in Docker
+1. Install Docker, and VSCode with Remote Extensions
+2. Clone this repo
+3. Open this repo in VSCode. When it offers to reopen in a container, say yes please.
+3. It'll fire up a local server to see your changes as you go. Look for
+your version on localhost:1313 (after about about a minute).
+5. If you want to make a new episode, open a terminal and run: `hugo new episode/my-episode-name.md` *Note - it is highly recommended that you follow this pattern rather than duplicating an existing file, as this will ensure you get the latest and greatest required frontmatter elements*
+
+### option 2: Docker outside of VSCode
+1. Install Docker
+2. Clone this repo
+3. If you want to fire up a local copy to see your changes as you go: `docker-compose up`
+4. You probably want to do the previous thing in another pane/window, as it needs to keep running.
+5. If you want to make a new episode, run a hugo command inside the docker container
+ that docker-compose started up: `docker exec ado-hugo_hugo_1 hugo new episode/my-episode-name.md` *Note - it is highly recommended that you follow this pattern rather than duplicating an existing file, as this will ensure you get the latest and greatest required frontmatter elements*
+
+### option 3: Install hugo
+1.  Install [Hugo](http://gohugo.io) - *NOTE: Please make sure it is at least version 0.58.3*
 2. Clone this repo
-3. If you want to fire up a local copy to see your changes as you go, enter this command `hugo server -w --baseUrl="http://localhost:1313"`
-4. You probably want to do the previous thing in another pane/window, as it needs to keep running. The -w watches for any changes and rebuilds on the fly. You have to set the baseUrl or it will build it with `www.arresteddevops.com`.
-5. If you want to make a new episode, type in `hugo new episode/my-episode-name.md` *Note - it is highly recommended that you follow this pattern rather than duplicating an existing file, as this will ensure you get the latest and greatest required frontmatter elements*
+3. If you want to fire up a local copy to see your changes as you go: `hugo server`
```

Comment:
Why was this command changed?

Reply:
the docker-compose sets an env var for the base URL, so you don't have to set it yourself.
(this was in the hugo image. not my bright idea, but useful)

Reply:
If you’re not using docker you have to keep this command as is. That’s what I am asking about - why it’s changed for the “not docker” section.

Reply:
oh shoot. yeah you're right, it should be the same for that